lgc/patch/PatchLoopMetadata.cpp

```diff
@@ -70,6 +76,43 @@ class PatchLoopMetadata : public Patch {
 // Initializes static members.
 char PatchLoopMetadata::ID = 0;
 
+// =====================================================================================================================
+// Update metadata by removing any existing metadata with the specified prefix, and then adding the new metadata if
+// existing metadata was removed or conditional is false.
+//
+// @param [in] loopID : loop
```

Comment:
loopID -> loopId

Reply:
I think you can remove [in] tag for all parameters in the comments. We only add [out] or [in/out]. All parameters are assumed to be input unless they are specially marked as output or inout.

Reply:
done